code/modules/admin/sql_message_system.dm

```diff
@@ -391,7 +450,7 @@
 			if(query_get_message_key.NextRow())
 				target_key = query_get_message_key.item[1]
 			qdel(query_get_message_key)
-		output += "<h2><center>[target_key]</center></h2><center>"
+		output += "<h2 style='font-size: 16px'><center>[target_key]</center></h2><center>"
```

Comment:
\>overriding h2 just to change the font size to be the same as h4
This is silly, don't do this.